### ISSUE TITLE
Unable to sync legacy edition worlds on xbox

### ISSUE THREAD

**Issue by hmm-web** [CrisXolt/Project-4JD-UI#50]
Since the import button doesnt work on xbox maybe it could be used to sync legacy edition worlds

**Comment by BestFurTH** [comment]
this is texture pack, not an actual legacy edition

**Comment by hmm-web** [comment]
> this is texture pack, not an actual legacy edition

I know.

**Comment by CrisXolt** [comment]
Yes, I noticed that I haven't added grid objects for legacy worlds yet, I'll figure it out later, don't worry. ;)
